5.1/5.1/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _5._1
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите текст: ");
+            string lineEng = Console.ReadLine();
+
+            string lineSymbolsandnumbers = LanguageLeet(lineEng);
+
+            Console.WriteLine("Перевод: " + lineSymbolsandnumbers);
+
+            Console.ReadKey();
+        }
+        static string LanguageLeet(string str)
+
+        {
+            str = str.Replace("A", "4");
+            str = str.Replace("B", "8");
+            str = str.Replace("C", "(");
+            str = str.Replace("D", "|)");
+            str = str.Replace("E", "3");
+            str = str.Replace("F", "|=");
+            str = str.Replace("G", "6");
+            str = str.Replace("H", "|-|");
+            str = str.Replace("I", "!");
+            str = str.Replace("J", ")");
+            str = str.Replace("K", "|<");
+            str = str.Replace("L", "1");
+            str = str.Replace("M", "|\/|");
```

Comment:
Знак \ в обыкновенной строковой константе означает начало управляющей последовательности (Esc-последовательности. Чтобы использовать \  в строковой константе как обыкновенный символ этот знак удваивают или используют буквальную строку (см. лекции по теме символы и строки).
```suggestion
            str = str.Replace("M", "|\\/|");
```
или
```suggestion
            str = str.Replace("M", @"|\/|");
```